lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -72,7 +72,7 @@ def os
   def detect_os
     return Specinfra.configuration.os if Specinfra.configuration.os
 
-    backend = Specinfra.backend
+    Specinfra.backend
```

Comment:
I wonder why this is here. `backend` has some side effect (validating the backend is valid) but I don't really see why we need to trigger those here. If we used it then it would have a caching benefit.
```suggestion
```